components/banner/BossBanner.tsx

```diff
@@ -0,0 +1,22 @@
+"use client"
+
+import { Banner } from "@bitcoin-dev-project/bdp-ui"
+
+const BossBanner = () => {
+    return (
+        <div className={`w-full dark:bg-black bg-white -z-20 sticky`}>
```

Comment:
negative z-index on <BossBanner /> why is that?

Reply:
for the banner, positive z-index doesn't work on it and even a -z-10 still makes the fixed decoding be above it.

i didn't want to change much on decoding because the banner is only temporary

Reply:
negative z-index forces it to be underneath the z stack. Also I noticed the banner is under the `header` component so the header styles apply on top of it.
That is you don't need to specify a z-index or sticky class as the header already has it

Reply:
<img width="1440" alt="Screenshot 2024-12-03 at 23 49 10" src="https://github.com/user-attachments/assets/a6e08776-a8f8-45b8-a2e3-0c6bb2433c19">
Yes but remember that the since the header is z-50 we also have the dialogue popover which stands under the main menu as since in the screenshot

Reply:
okay, that makes sense then 👍